extensions/mssql/package.json

```diff
@@ -485,6 +491,28 @@
             }
           ]
         }
+      },
+      {
+        "id": "databases-tab",
+        "description": "%databases-tab-description%",
+        "provider": "MSSQL",
+        "title": "%databases-tab-title%",
+        "when": "",
```

Comment:
I need to figure out the when expression for it to only show for server context. i will have another commit.

Reply:
fixed